stdlib/public/runtime/ImageInspectionCOFF.cpp

```diff
@@ -42,10 +43,55 @@ void record(const swift::MetadataSections *sections) {
 }
 }
 
+#ifndef NDEBUG
+SWIFT_RUNTIME_EXPORT
```

Comment:
The COFF & ELF implementations of this logic look identical. Is there a way to factor out the code into a shared header?